tests/e2e/utils/flows.js

```diff
@@ -276,7 +276,44 @@ export const merchantWCP = {
 
 		if ( ! ( await page.$( '#_wcpay_feature_upe:checked' ) ) ) {
 			await expect( page ).toClick( 'label', {
-				text: 'Enable UPE checkout',
+				text: 'Enable UPE checkout (legacy)',
+			} );
+		}
+
+		const isSplitUPEEnabled = await page.$(
+			'#_wcpay_feature_upe_split:checked'
+		);
+
+		if ( isSplitUPEEnabled ) {
+			await expect( page ).toClick( 'label', {
+				text: 'Enable Split UPE checkout',
+			} );
+		}
```

Comment:
This E2E flow may require Automattic/woocommerce-payments-dev-tools#80 to be merged first.